src/ScottPlot5/ScottPlot/Colormaps/GrayscaleReversed.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace ScottPlot.Colormaps
+{
+    public class GrayscaleReversed : IColormap
+    {
+        public string Name => "Grayscale Reversed";
+
+        public Color GetColor(double intensity, Range? domain)
+        {
+            if (double.IsNaN(intensity))
+            {
+                return Colors.Transparent;
+            }
+
+            domain ??= Range.UnitRange;
+
+            double normalized = domain.Value.NormalizeAndClampToUnitRange(intensity);
```

Comment:
Same comment as above (wow, lots of duplicated code I guess)

Reply:
Yeah I introduced a `ColormapBase` to avoid some of that, I like the final result but I'm worried we're reaching Java-levels of OOP.